librz/core/cmd_analysis.c

```diff
@@ -6042,6 +6042,73 @@ static void cmd_analysis_ucall_ref(RzCore *core, ut64 addr) {
 	}
 }
 
+static RzFlagItem *core_flag_get_at_as_ref_type(RzCore *core, RzAnalysisXRef *xrefi) {
```

Comment:
What do you want to do exactly with this function? Is it something like `rz_flag_get_by_spaces` ?

Reply:
actually, it is. maybe iu should just replace that with the proper one.